homeassistant/components/sunsynkweb/coordinator.py

```diff
@@ -0,0 +1,56 @@
+"""Coordinator for the sunsynk web api."""
+
+from asyncio import timeout as async_timeout
+from datetime import timedelta
+import logging
+
+from pysunsynkweb.model import get_plants
+from pysunsynkweb.session import SunsynkwebSession
+
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class PlantUpdateCoordinator(DataUpdateCoordinator):
```

Comment:
```suggestion
class SunsynkUpdateCoordinator(DataUpdateCoordinator):
```
makes it clearer for those not knowing sunsynk, because it's weird initially we have a "Plant"

Reply:
fair enough, updating